examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
+sudo pkill -x openshift
```

Comment:
@ncdc ran into issues with similar logic killing the Docker daemon since it's command-line invocation had the OpenShift registry as a `--insecure-registry` set up. Will this have the same issues?